.github/ISSUE_TEMPLATE/enhancement.md

```diff
@@ -4,7 +4,10 @@ about: Suggest an enhancement to the Kubernetes project
 labels: kind/feature
 
 ---
-<!-- Please only use this template for submitting enhancement requests -->
+<!-- Feature requests are unlikely to make progress as an issue.
+
+Instead, please suggest enhancements by following the KEP process: https://git.k8s.io/enhancements/keps#kubernetes-enhancement-proposals-keps
+-->
```

Comment:
```suggestion
<!-- 
ATTENTION: Enhancement requests will NOT make progress as an issue.

Instead, reach out and propose your feature to the relevant Special Interest Group (SIG) on their Mailing List or Slack: https://k8s.dev/groups

For more information, read more about the Kubernetes Enhancement Proposal Process:
https://git.k8s.io/enhancements/keps#kubernetes-enhancement-proposals-keps
-->
```

The current KEP documentation isn't going to be helpful to most folk =/ I think directing them to reach out to the mailing list or in slack would provide better direction.

I would also change the about text, and potentially the name itself.

```
name: Enhancement Tracking Issue
about: Provide supporting details for a feature in development
```
I'm not super keen on the wording above, but I think between changing that and the text at the top we can steer more people to the right place.

Reply:
Done, PTAL.